web/src/settings_ui.ts

```diff
@@ -6,46 +6,63 @@ import {$t, $t_html} from "./i18n";
 import * as loading from "./loading";
 import * as ui_report from "./ui_report";
 
-export function display_checkmark($elem) {
+export function display_checkmark($elem: JQuery<HTMLElement>): void {
     const check_mark = document.createElement("img");
     check_mark.src = checkbox_image;
     $elem.prepend(check_mark);
     $(check_mark).css("width", "13px");
 }
 
-export const strings = {
+type Strings = {
+    failure_html: string;
+    saving: string;
+    success_html: string;
+};
+
+export const strings: Strings = {
```

Comment:
This isn’t a function, and this type annotation doesn’t serve any purpose other than to duplicate a bunch of information. We can just let TypeScript infer this type.

Reply:
Okay